jsonld.js/src/main/scala/org/w3/banana/jsonldjs/js/JsonLdJs.scala

```diff
@@ -0,0 +1,99 @@
+package org.w3.banana
+package jsonldjs
+package js
+
+import scala.scalajs.js
+import scala.concurrent.{ Future, Promise }
+
+// the content of foo should be in jsonld but it doesn't work! Don't
+// know why...
+object foo {
```

Comment:
I did in https://github.com/w3c/banana-rdf/commit/669dd3955979e4cd222326267006f3696c952b47 but couldn't make it work. So I just renamed foo in jsonldHelper for now.